xmtp_id/src/associations/unsigned_actions.rs

```diff
@@ -14,58 +19,52 @@ pub struct UnsignedCreateInbox {
 
 impl SignatureTextCreator for UnsignedCreateInbox {
     fn signature_text(&self) -> String {
-        format!(
-            // TODO: Finalize text
-            "Create Inbox: {}",
-            generate_inbox_id(&self.account_address, &self.nonce)
-        )
+        format!("- Create inbox\n(Owner: {})", self.account_address)
```

Comment:
```suggestion
        format!("- Create inbox\n  (Owner: {})", self.account_address)
```
Small nit to get alignment - added two whitespaces